client/src/components/404/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './styles.css';
+
+const NotFound = (props) => {
+  const message = 'It looks like you\'re lost...';
+
+  return (
+    <div className="not-found-container">
+      <h1 className="title">404</h1>
+      <div className="picture" />
```

Comment:
Почему не тег img?

Reply:
Because I do not want it to be draggable
When you drag img element you can see transparent image and I do not like it

Reply:
Just do not drag it. What is the problem?
But if you really do not like this behavior you can set `pointer-events: none;` on img element.

Reply:
Ahaha, I do not like the way you think!

- I said this according to the current example, not in general.
- How can app breaks when you drag an image?
- Almost all content on the page is draggable.

In this case image is a part of content.
And for this case using `img` element is appropriate.
Let's imagine... Some user liked your image and wants to save it.
He can't do this if image is set as you did.
Or another scenario...
For screen readers `div` without content will be an empty element, without any sense for the people who can't see.

There are a lot of html tags and each of them created with certain purpose.
You can write your code using only divs and it will work.
But for some people your website will be hard to read and code would be with smells :)
The interface should be created correctly.

Reply:
Do not take personally my joke about the button. My point was that if I do not like that when someone drags the image it becomes visible - then "Just do not drag it. What is the problem?" is not the best solution.

Also, thank you for the great reasoning about why you think the `img` tag is better than `div`. I really appreciate this. Some points from your reasoning I was aware of and these are my points why I decided to use `div` and not `img`:
- from my perspective in this case the image is NOT a part of the content, it is just a background picture a wallpaper on the desktop screen. This is why I did not want it to be draggable. 
- "If the user liked the picture and wants to save it" - I do not really care about this case because my goal is that this picture was present on my 404 page, not to distribute it around the world. If someone really needs this pic - they can search it in google or use DevTools of the browser.
- "For screen readers div without content will be an empty element" - it has a class "picture" so I think for blind people it will be as useful as to know that there is an img tag.

"The interface should be created correctly." - I am totally agree, but sometimes there is no very strict rule what is better and what is worse. In this case I feel like `div` is more suitable... ` ¯\_(ツ)_/¯`

Reply:
It is not a strict rule. But some case of best practice :)

"it has a class "picture"" - screen readers does not use class attribute.
Therefore `<div className="picture" />` just an empty element.
Even if you use img element you should provide alt attribute.
https://www.powermapper.com/tests/screen-readers/attributes/